src/fsharp/FSharp.Core/printf.fs

```diff
@@ -951,7 +948,7 @@ module internal PrintfImpl =
                 arr.[start + i] <- s.Pop()
             arr
         
-        member this.GetArgumentAndTypesAsArrays
+        member __.GetArgumentAndTypesAsArrays
```

Comment:
@forki @dsyme 
I am not a fan of this change,  I know you have used it in other cleanup PRs, however, I don't recall a discussion where this was the agreed style.

If we were to agree to a common style for this my preference would be in fact be 'this', since that is what it in fact is.  I assume that you want to indicate that the this pointer is unused, but that is not an entirely interesting distinction, given that we are not writing in C# and we are immutable by default.  Anyway ... let's have a discussion about a coding standard for this case before we carry on with the clean up.

Kevin